Siri/L1/Level1.java

```diff
@@ -0,0 +1,19 @@
+import java.util.Scanner;
+
+public class Level1 {
```

Comment:
Perhaps adding a Javadoc for each class and public method?

Reply:
I agree that adding a Javadoc for each class and public method would allow the programme to be better understood